File: src/main/java/no/nav/pensjon/regler/domain/beregning2011/AfpKronetillegg.java
package no.nav.pensjon.regler.domain.beregning2011;

import no.nav.pensjon.regler.domain.beregning.Ytelseskomponent;
import no.nav.pensjon.regler.domain.kode.YtelsekomponentTypeCti;

public class AfpKronetillegg extends Ytelseskomponent {
    private static final long serialVersionUID = -5451157372553547398L;

    public AfpKronetillegg() {
        super();
        ytelsekomponentType = new YtelsekomponentTypeCti("AFP_KRONETILLEGG");
    }

    public AfpKronetillegg(AfpKronetillegg afpKronetillegg) {
        super(afpKronetillegg);
    }

}

File: src/main/kotlin/no/nav/pensjon/regler/domain/beregning2011/AfpKronetillegg.kt
package no.nav.pensjon.regler.domain.beregning2011

import no.nav.pensjon.regler.domain.beregning.Ytelseskomponent
import no.nav.pensjon.regler.domain.enum.YtelseskomponentTypeEnum

class AfpKronetillegg : Ytelseskomponent() {
    override val ytelsekomponentTypeEnum: YtelseskomponentTypeEnum = YtelseskomponentTypeEnum.AFP_KRONETILLEGG
}